samplesheets/views.py

```diff
@@ -2538,37 +2561,42 @@ class IrodsRequestAcceptView(
     IrodsRequestModifyMixin,
     FormView,
 ):
-    """View for accepting an iRODS data request"""
+    """View for accepting iRODS data requests"""
 
     permission_required = 'samplesheets.manage_sheet'
     template_name = 'samplesheets/irods_request_accept_form.html'
     form_class = IrodsRequestAcceptForm
 
+    def get_form_kwargs(self):
+        """Override to pass number of requests to form"""
+        kwargs = super().get_form_kwargs()
+        kwargs['num_requests'] = 1
+        return kwargs
+
     def get_context_data(self, *args, **kwargs):
         context_data = super().get_context_data(*args, **kwargs)
         obj = IrodsDataRequest.objects.filter(
             sodar_uuid=self.kwargs['irodsdatarequest']
         ).first()
+        context_data['irods_request_data'] = []
```

Comment:
`irods_request_data` seems vague. Could this be `irods_requests`, or maybe `request_objects`?